TypeScript.py

```diff
@@ -439,6 +445,49 @@ def checkUpdateView(view):
         if cli.reloadRequired(view):
             reloadBuffer(view, clientInfo)
 
+# these three methods work with signature help
+def handle_signature_help(signature_help_response):
+    if signature_help_response.success:
```

Comment:
Also we're very inconsistent about casing throughout, consider renaming the parameter sy least to match its locals.